### DISCUSSION TITLE
Register interest for: Pixelmon server list

### DISCUSSION THREAD

**Discussion by Brend-Smits** [ServerListPlus/issues#15] [General]
Please comment down below to register your interest in a Pixelmon server list. 
We are working on one, but need to know how many of you are wanting to use it so we know what features to focus on and how fast we should look into deploying a Pixelmon version.   
  
This is also an easy way to stay updated on when it's released.

Template to use to register interest:
```
Hey, I'm Interested,
**Server name:**
```

**Update 21/05/2022:**
Hey there all!

The [Pixelmon-Servers](https://pixelmon-servers.com/) server list has now been released. Please read the release post and notes here: [v1.4.2 (release)](https://github.com/ServerListPlus/issues/releases/tag/v1.4.2)

Thank you for your interest and for your patience. This has taken quite a bit of time, but we wanted to get it right. We have some cool features coming up as well, read more about them in the release notes!
Feel free to share this news with others. Let me know if you need any help setting up.

**Comment by austin4195** [comment]
Hey, I'm Interested,
**Server name: PokeHills**

**Reply by Brend-Smits** [reply]
Thanks, Austin, we will keep you updated ^.^ If you have any other feature suggestions, questions or concerns, please don't hesitate to ask.

**Comment by VOrlando520** [comment]
Hey, I'm Interested,
PokeOdyssey

**Reply by Brend-Smits** [reply]
Thank you, Vince, let me know if you have any feature suggestions, questions, or concerns!

**Comment by soggyhawk** [comment]
Hey, I'm Interested,
Server name: Argon Gaming

**Comment by Brend-Smits** [comment]
We're getting close to the point where we feel comfortable releasing a Pixelmon version of our server list! In the coming week(s) we will be adding a few more features, then we will look into planning a release date for https://pixelmon-servers.com 
Thank you all for your interest, if you know more server owners that might be interested, please share this link.

**Comment by Brend-Smits** [comment]
Hey there all! 

The [Pixelmon-Servers](https://pixelmon-servers.com) server list has now been released. Please read the release post and notes here: https://github.com/ServerListPlus/issues/releases/tag/v1.4.2

Thank you for your interest and for your patience. This has taken quite a bit of time, but we wanted to get it right. We have some cool features coming up as well, read more about them in the release notes!
Feel free to share this news with others. Let me know if you need any help setting up.

cc @soggyhawk, @VOrlando520, @austin4195
